src/main/java/com/contentgrid/gateway/runtime/config/kubernetes/KubernetesLabels.java

```diff
@@ -0,0 +1,11 @@
+package com.contentgrid.gateway.runtime.config.kubernetes;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+class KubernetesLabels {
```

Comment:
If it contains some constants, `interface` works equally well.

And then at least the constants are final. Now they are modifiable static fields.

Reply:
Hah, they were an interface first. Sonarcube calls that a **major** code smell. Something about not the intentional usage of interfaces. But I should definately make them `final` and `static`

Reply:
`@UtilityClass` makes them static anyways, but not final of course.